src/main/java/game/baseball/util/FixedNumberGenerator.java

```diff
@@ -0,0 +1,31 @@
+package game.baseball.util;
+
+import java.util.List;
+import java.util.Optional;
+
+import static game.baseball.game.GameSetting.GAME_SETTING_VERSION_01;
+import static game.baseball.view.input.InputHandler.*;
+
+public class FixedNumberGenerator implements NumberGenerator {
+
+    @Override
+    public List<Integer> generate() {
+        return Optional
+                .ofNullable(getFixedNumbers())
+                .orElseThrow(() -> new NumberFormatException("올바르지 않은 포맷입니다.")); // TODO : Exception 관리
+    }
+
+    private List<Integer> getFixedNumbers() {
+        List<Integer> fixedNumbers = convertToIntegerList(splitBySpace(readLine()));
+        if (!checkNumberSize(fixedNumbers)) {
+            fixedNumbers = getFixedNumbers();
+        }
+        return fixedNumbers;
+    }
+
+    private boolean checkNumberSize(List<Integer> fixedNumbers) {
+        if (fixedNumbers.size() == GAME_SETTING_VERSION_01.getBallSize()) return true;
+        return false;
+    }
```

Comment:
```suggestion
    private boolean checkNumberSize(List<Integer> fixedNumbers) {
        return fixedNumbers.size() == GAME_SETTING_VERSION_01.getBallSize()
    }
```

이런식으로 작성하는 것이 훨씬 가독성도 높고 굳이 쓸데 없는 코드들이 들어가지 않아도 될거 같습니다.